playbooks/byo/openshift-cluster/upgrades/v4_0/upgrade_nodes.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I don't know why we still have the byo plays, why add this here now?

Reply:
IIUC tox tests want these

Lets see what happens if I remove those

Reply:
oh, I forgot to rename this one. Fixed